custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
           {
-            "$ref": "#/parameters/listChatThreadsStartTime"
+            "in": "query",
+            "name": "language",
+            "description": "The language code is in BCP 47 format.",
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/listChatThreadsMaxPageSize"
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/InvoiceDocumentRequest"
+            }
           }
-        ]
-      }
-    },
-    "/chat/threads/{chatThreadId}/participants": {
-      "get": {
+        ],
         "responses": {
           "200": {
-            "description": "Success.",
+            "description": "The request has succeeded.",
             "schema": {
-              "type": "object",
-              "properties": {
-                "value": {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "properties": {
-                      "communicationIdentifier": {
-                        "$ref": "#/definitions/CommunicationIdentifier"
-                      },
-                      "displayName": {
-                        "x-ms-summary": "Name",
-                        "type": "string",
-                        "description": "Display name for the chat participant."
-                      },
-                      "shareHistoryTime": {
-                        "x-ms-summary": "Time",
-                        "type": "string",
-                        "description": "Time in RFC3339 format (yyyy-MM-ddTHH:mm:ssZ) from which the chat history is shared."
-                      }
-                    }
-                  },
-                  "description": "Chat participants."
-                },
-                "nextLink": {
-                  "type": "string",
-                  "description": "If there are more chat participants that can be retrieved, the next link will be populated.",
-                  "x-ms-summary": "Next Link"
-                }
+              "format": "binary",
+              "type": "string"
+            },
+            "headers": {
+              "Content-Type": {
+                "type": "string",
+                "enum": [
+                  "application/pdf",
+                  "application/zip"
+                ]
+              },
+              "Content-Disposition": {
+                "type": "string"
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "List chat thread participants",
-        "description": "List participants in chat thread.",
-        "operationId": "ListParticipants",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/ListParticipants

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.